### PR TITLE
th-46: Create Loading component

### PR DESCRIPTION
**Pull request by AlexVlaso**
Снимок экрана 2023-08-22 в 20 50 50

### CODE REVIEW

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import style from './spinner.module.scss';
```
Comment (kalinkaaaa14) [inline frontend/src/libs/components/spinner/spinner.tsx]:
```suggestion
import styles from './styles.module.scss';
```

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (kalinkaaaa14) [inline frontend/src/libs/components/spinner/spinner.tsx]: no need to use it

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <div className={style.wrapper}>
+      <div className={style.container}>
+        <div className={style.loader}>Loading...</div>
+      </div>
+    </div>
+  ) : (
+    <div className={style.container}>
+      <div className={style.loader}>Loading...</div>
+    </div>
+  );
```
Comment (kalinkaaaa14) [inline frontend/src/libs/components/spinner/spinner.tsx]: use early return here
if(isFullScreen ) { return one jsx; } return default jsx;

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import styles from './spinner.module.scss';
```
Comment (kalinkaaaa14) [inline frontend/src/libs/components/spinner/spinner.tsx]:
```suggestion
import styles from './styles.module.scss';
```

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import styles from './styles.module.scss';
+
+type Properties = {
+  isFullScreen?: boolean;
```
Comment (Druxight) [inline frontend/src/libs/components/spinner/spinner.tsx:6]: Add size configuration (sm, md, lg) 
default md

**Review by Druxight — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/spinner/spinner.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  size?: 'sm' | 'md' | 'lg';
+};
+
+const Spinner: React.FC<Properties> = ({
+  isFullScreen,
+  size = 'sm',
+}: Properties) => {
+  const loaderClassName = `${styles.loader} ${styles[size]}`;
```
Comment (Druxight) [inline frontend/src/libs/components/spinner/spinner.tsx]: Use clsx (its already in the dev branch)

**Review by kalinkaaaa14 — APPROVED** [review]

**Review by Druxight — APPROVED** [review]

**Pull request merged**